core/src/main/java/org/jdbi/v3/core/locator/ClasspathSqlLocator.java

```diff
@@ -75,7 +79,11 @@ private static String resourcePathFor(String fullyQualifiedName) {
     }
 
     private static String findSqlOnClasspath(ClassLoader classLoader, String path) {
-        return CACHE.computeIfAbsent(path, p -> readResource(classLoader, p));
+        try {
+            return CACHE.get(path, () -> readResource(classLoader, path));
```

Comment:
Oh, FFS Guava. You can't use a lambda because it's an abstract class. Because of course it is.